test/leafTest.js

```diff
@@ -0,0 +1,151 @@
+const { BigNumber } = require("ethers")
+const { expect } = require("chai")
+
+describe("Leaf", () => {
+  let leafInstance
+
+  beforeEach(async () => {
+    const LeafStub = await ethers.getContractFactory("LeafStub")
+    leafInstance = await LeafStub.deploy()
+    await leafInstance.deployed()
+  })
+
+  describe("make", async () => {
+    it("constructs a leaf", async () => {
+      // We store a leaf as a uint256, which is 256 bits. We pack the address
+      // in the first 160 bits, the creation block in the 64 bits after that,
+      // and the operator's id in the final 32 bits. Each hex character
+      // represents 4 bits, so a 40-character hex representation of an eth
+      // address is the exact 160 bits we need. A number (like a creation block
+      // or id) gets left-padded with zeros to fill the exact bit count.
```

Comment:
Worth mentioning which byte endianness is used in order to be super explicit.

Reply:
[3fc902e](https://github.com/keep-network/sortition-pools/pull/173/commits/3fc902e6a156576522881b07f3c2de1b5cb51773)

Reply:
Hmm, I'm not sure if little-endian is correct. Please see https://github.com/ethereum/solidity-examples/issues/54

Reply:
[2c6b136](https://github.com/keep-network/sortition-pools/pull/173/commits/2c6b136aeab7aab43e9c7ebb8c10a5acb66e824f)